consultation_analyser/consultations/public_schema/consultation_example.json

```diff
@@ -1 +1 @@
-{"name":"How should Kit Kats change","sections":[{"id":"4a52c1c9-0131-cf3a-838c-9cf0a8b98dea","name":"When Kit Kats are consumed","questions":[{"id":"743be937-a7c8-2cd2-b7da-b5e06d06e440","text":"Should it happen on a full moon?","has_free_text":false},{"id":"a7f50741-0eb9-e569-e7c9-defbe544ce60","text":"Should it happen on Fridays?","has_free_text":true},{"id":"53d4a965-e746-ebc3-6608-417b0d74393f","text":"Should it happen on Fridays?","has_free_text":false},{"id":"c1f7b211-088d-c952-1bd5-09ce76c69adc","text":"Should it happen on a full moon?","has_free_text":true}]}]}
+{"name":"Consultation on Kit Kats","sections":[{"id":"81f6a219-96fc-c964-135e-c015d4021065","name":"When to enforce a Kit Kat ban","questions":[{"id":"a7ee1898-4b3e-150a-dc9d-7aece43eb8fc","text":"Should it happen in the month of May?","has_free_text":false}]}]}
```

Comment:
If this is the format we're asking for the data in, the users won't have IDs. Should these be removed?